### PR TITLE
Show who has User admin role in User list [#2619]

### PR DESCRIPTION
**Pull request by dlebrero**
#### Before the PR (what is the issue or what needed to be done)

See #2619 

#### The solution

See #2619 

Reusing same "admin?" logic as in the edit user checkbox

## Checklist
* [x] Connect the issue
* [ ] Test plan
* [x] Copyright header
* [x] Code formatting
* [ ] Documentation

### CODE REVIEW

File: Dashboard/app/cljs/src/org/akvo/flow/dashboard/users/users_list.cljs [hunk 1]
```diff
@@ -98,9 +98,9 @@
                            {:user user
                             :user-auth-store user-auth-store
                             :roles-store roles-store})}
-     {:title (t> _api_keys)
-      :class "text-center"
-      :component api-user-mark}
+     {:title     (t> _admin)
```
Comment (iperdomo) [inline Dashboard/app/cljs/src/org/akvo/flow/dashboard/users/users_list.cljs]: There is another string using the `_administrator` translation key, why do we have 2?

Comment (dlebrero) [reply Dashboard/app/cljs/src/org/akvo/flow/dashboard/users/users_list.cljs]: That key is:

_administrator = Administrator - Create, Modify and Delete Users

which is very long for that column.

Comment (iperdomo) [reply Dashboard/app/cljs/src/org/akvo/flow/dashboard/users/users_list.cljs]: You're right. Then let's _fix_ the value of the new translation key to: `User Admin` ... not all upper case. The uppercase text transformation is done in the UI via CSS.

Comment (dlebrero) [reply Dashboard/app/cljs/src/org/akvo/flow/dashboard/users/users_list.cljs]: done

**Comment by dlebrero** [comment]
Merged as part of https://github.com/akvo/akvo-flow/pull/2689

**Pull request closed without merging**